attribute/parser.go

```diff
@@ -46,7 +46,10 @@ func parseValue(contentType yotiprotoattr.ContentType, byteValue []byte) (interf
 
 	case yotiprotoattr.ContentType_JPEG,
 		yotiprotoattr.ContentType_PNG:
-		return byteValue, nil
+		return ParseImageValue(contentType, byteValue)
+
+	case yotiprotoattr.ContentType_UNDEFINED:
```

Comment:
I think it would be good to add a bit of logging here, I think we have that in the other SDKs.
Something like:
```go
log.Print("UNDEFINED Content type, attempting to parse as a string")
```

Reply:
If we add the logging, then what's the difference between UNDEFINED and default?

Reply:
UNDEFINED has this comment in Attribute.proto:
> `UNDEFINED should not be seen, and is used as an error placeholder value.

So if something went wrong, it will be UNDEFINED, where as when it is a new type which is not recognized (yet), then it will be default